android-kit-base/src/test/kotlin/com/mparticle/kits/KitManagerImplTest.kt

```diff
@@ -1105,10 +1105,11 @@ class KitManagerImplTest {
             String::class.java,
             MParticleUser::class.java,
             IdentityApi::class.java,
+            KitConfiguration::class.java,
             Runnable::class.java
         )
         method.isAccessible = true
-        val result = method.invoke(manager, "Test@gmail.com", "", user, identityApi, runnable)
+        val result = method.invoke(manager, "Test@gmail.com", "", user, identityApi, mockedKitConfig, runnable)
```

Comment:
I was writing this comment before we hopped on with Brandon, but it will still apply once the additional updates with an `Unassigned` value are included.

Can you add a test where the mockedkitconfig has a setting for `hashedEmailUserIdentityType` of something besides `other`?  Then the assertion would show that it makes a request with that identity?